app/javascript/styles/mastodon/components.scss

```diff
@@ -6627,15 +6627,20 @@ noscript {
       margin-top: 5px;
     }
 
+    &__buttons {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      margin-top: 5px;
+    }
```

Comment:
*(Clears throat)*, what do you think... of this too:

```diff

  .account__header__user-note__buttons {
-   justify-content: space-between; */
+   justify-content: flex-end;
  }

+  .account__header__user-note__buttons > button {
+    margin-left: 15px;
+ }

```

`space-between` seems radical on  desktop, when one button here, other there. This moves them closer on the right (which ensures small pointer travel time from edit button), and margin ensures they spaced a little (I don't know of any other ways to replicate that).